.github/workflows/rerun.yml

```diff
@@ -0,0 +1,25 @@
+# Cross version tests sometimes fail due to transient errors. This workflow reruns failed tests.
+name: Rerun
```

Comment:
Maybe change to "Rerun Cross Version Test"? We are just rerunning the cross version tests.

Reply:
sounds good.